stl/src/xdateord.cpp

```diff
@@ -11,7 +11,7 @@ _EXTERN_C_UNLESS_PURE
 

```

Comment:
There are also some occurrences of `sizeof / sizeof` in `stl/inc` that can be updated, although several of them need `static_cast` to avoid truncation warnings (someday, `consteval` would allow the compiler to see that no truncation is possible, but not today). I'll push changes.